build/platform-id.mk

```diff
@@ -333,13 +333,13 @@ PLATFORM_LWIP=posix
 endif
 
 ifeq ("$(PLATFORM_ID)","22")
-PLATFORM=argon-som
+PLATFORM=asom
 PLATFORM_NAME=argon
```

Comment:
This is used in https://github.com/particle-iot/device-os/blob/develop/hal/src/nRF52840/device_code.cpp#L93, so despite all the changes the broadcast BLE name will still remain (Argon-XXXXXX/Boron-XXXXXX/Xenon-XXXXXX for SoMs).

Reply:
Thank @avtolstoy :) @monkbroc do we want to broadcast a different name than Argon-XXXXXX, etc.. for SoMs?  ASoM-XXXXXX seems a little weird without the space, and I think the space would break things.

Reply:
That's fine. Leave it as is for now as it would break mobile apps otherwise. We can change it later if needed.

Reply:
We are going to be reworking the mobile apps for the upcoming BLE APIs and changes to how setup mode functions, so there is scope there to rework the apps when we decide on the final name.